### DISCUSSION TITLE
Reinstalled Rectangle Pro via Brew and settings are not synced via iCloud

### DISCUSSION THREAD

**Discussion by honeybyhomo** [rxhanson/RectanglePro-Community#415] [Q&A]
Hi,
I'm moving all my apps to Brew:
1. Made sure "sync via iCloud" was enabled.
2. Deleted Rectangle Pro
3. Installed it from Brew `brew install rectangle-pro`.
4. Enabled "sync via iCloud".
5. Activated the license.
But nothing is synced, Rectangle Pro is still in it's "default" settings.
Any idea how to get it to sync?

Thanks!

**Comment by rxhanson** [comment]
Hmm. I suspect this might have to do with using brew. I’ll have to test some things out to confirm.

**Reply by honeybyhomo** [reply]
Thanks. What can I do (manually) to sync/import everything I've made previously?

**Reply by rxhanson** [reply]
There are buttons in the general tab of the setting window for exporting and importing your config as json.

**Reply by honeybyhomo** [reply]
Since I have already uninstalled and installed via `brew install`, where can I find the config files (on iCloud)?

**Reply by rxhanson** [reply]
As far as I know, there's not a way to manually pull this out of iCloud, unfortunately. I think everything just has to be lined up nicely for iCloud sync to work, so the other option you could try is 
1. uninstall the brew one
2. reinstall the dmg one
3. export the config
4. uninstall then reinstall from brew
5. import config

**Reply by Aaron-Rumpler** [reply]
All `brew install rectangle-pro` should have done is downloaded the DMG and copied `Rectangle Pro.app` from it into `/Applications`. Assuming you didn't touch the config files in `~/Library`, it _should_ have just opened with your old config (including the old activation).

**Reply by Aaron-Rumpler** [reply]
Another option would be to pull `~/Library/Preferences/com.knollsoft.Hookshot.plist` from a recent backup if you have one.

**Comment by Aaron-Rumpler** [comment]
Regarding your migration to Homebrew casks, have you tried `brew install --adopt`? It checks to see if the version of the app currently installed is the same as what `brew install` would install, and if so marks that cask as installed.

**Reply by honeybyhomo** [reply]
No, but that would probably have been the smart solution :-) I have already uninstalled and installed via `brew install` so that's not an option anymore.
